saleor/graphql/payment/tests/mutations/test_payment_gateway_initialize.py

```diff
@@ -860,3 +860,98 @@ def test_with_payment_gateways_and_amount_with_lot_of_decimal_places(
         ],
         order,
     )
+
+
+@mock.patch("saleor.webhook.transport.synchronous.transport.send_webhook_request_sync")
+@override_settings(PLUGINS=["saleor.plugins.webhook.plugin.WebhookPlugin"])
+def test_for_checkout_with_shipping_app(
```

Comment:
IMO we should also add a test for tax webhook.

Reply:
Would be also good to have a test for case when order is a `sourceObject`

Reply:
Added both cases, please re-review 🙏